README.md

```diff
@@ -1,10 +1,5 @@
-# SomaLogic-Data
+# SomaLogic-Data <img src="figures/logo.png" align="right" height="138" alt=""/>
```

Comment:
Is this a legit file location? I don't see `figures/logo.png` in this repository. Perhaps you need to add it? 

Shoot, now I see you've added it below ... could you ensure that the logo is the correct one (R), as used in the other repos?